### ISSUE TITLE
`fromEmail` and `fromLabel` cause an error when creating an email campaign

### ISSUE THREAD

**Issue by hevele-moda** [klaviyo/klaviyo-api-node#67]
Try creating an email campaign using the Node SDK (version 10.0.0):
```
import { Campaigns } from 'klaviyo-api';

Campaigns.createCampaign({
  data: {
    type: 'campaign',
    attributes: {
      name: 'Email',
      audiences: {
        included: []
      },
      sendStrategy: {
        method: 'static',
        optionsStatic: {
          datetime: new Date()
        }
      },
      campaignMessages: {
        data: [
          {
            type: 'campaign-message',
            attributes: {
              channel: 'email',
              content: {
                subject: 'Hello',
                fromEmail: 'foo.bar@example.com',
                fromLabel: 'Foo Bar'
              }
            }
          }
        ]
      }
    }
  }
});
```

It returns an error saying that `fromEmail` and `fromLabel` are not accepted:
```
{
  "id": "4ac9f851-6027-4c51-9023-eeb7a57592dc",
  "status": 400,
  "code": "invalid",
  "title": "Invalid input.",
  "detail": "'fromEmail' is not a valid field for the resource 'content'.",
  "source": {
    "pointer": "/data/attributes/campaign-messages/data/0/attributes/fromEmail"
  },
  "links": {},
  "meta": {}
}
```
```
{
  "id": "855fbe4d-47c4-4481-a383-df9a6df32d86",
  "status": 400,
  "code": "invalid",
  "title": "Invalid input.",
  "detail": "'fromLabel' is not a valid field for the resource 'content'.",
  "source": {
    "pointer": "/data/attributes/campaign-messages/data/0/attributes/fromLabel"
  },
  "links": {},
  "meta": {}
}
```

**Comment by Ian-Montgomery-Klaviyo** [comment]
There was a bug in the `ObjectSerializer` class, the fix is in the newly released version `10.0.1`. If you update the issue should resolve.

**Comment by hevele-moda** [comment]
> There was a bug in the `ObjectSerializer` class, the fix is in the newly released version `10.0.1`. If you update the issue should resolve.

Thank you!
